packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -82,6 +82,7 @@ export class SignUp extends BaseResource implements SignUpResource {
         paramsWithCaptcha.captchaToken = captchaToken;
         paramsWithCaptcha.captchaWidgetType = captchaWidgetTypeUsed;
       } catch (e) {
+        console.error('Clerk CAPTCHA Error: ', e.captchaError || e.message || e);
```

Comment:
Not a big fan of this, but if we have a big issue and this will help devs with debugging, then i'm ok with it.

Reply:
Not a big fan either, but I added this because in some cases we replace the actual error value with a generic error message. We could remove this of course.

Reply:
So, I'll remove this one and leave only the specific one about the script failing.